kart/status.py

```diff
@@ -100,8 +105,33 @@ def status(ctx, output_format):
 
     if output_format == "json":
         dump_json_output({"kart.status/v2": jdict}, sys.stdout)
-    else:
+    else:       
         click.echo(status_to_text(jdict))
+        
+        if list_untracked_tables:
+            """Check for any untracked tables in working copy"""
+            wc = repo.working_copy.tabular 
+            click.echo(wc)
+            if wc is not None:
+                if wc.session() is not None:        
+                    with wc.session() as sess: 
+                        wc_items = wc.adapter.list_tables(sess)
+                    # Get all tables in working copy
+                    all_tables = [table_name for table_name, title in wc_items.items()]
+                    # Get tables shown in kart data ls
+                    datasets_paths = [datasets.path for datasets in repo.datasets()] 
```

Comment:
Two comments here:
1. Use the attribute `dataset.table_name` - a dataset's path is allowed to be slightly different to its table name, eg the dataset with path `aaa/bbb` is checked out into a table called `aaa__bbb` I think
However: not all datasets have a table_name attribute, so, use repo.datasets(filter_dataset_type="table") to make sure you only get the table datasets which have this attribute.
2. (Nitpick) `datasets_paths = [datasets.path for datasets in repo.datasets()]` - the variable `datasets` should be `dataset` since it is singular - it points to a single dataset (at a time)